server/controllers/user.js

```diff
@@ -0,0 +1,122 @@
+const usersModel = require("../model/user")
+const express = require ("express");
+const bodyParser = require("body-parser");
+
+const app = express();
+
+const Logger = require('../services/logger_services'); 
+const logger = new Logger('app'); 
+
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(bodyParser.json());  
+
+
+const idGeneretor = {
+    lastId: 0,
+    generateId: function() {
+        return ++(this.lastId);
+    },
+};  
+
+
+const getUser = (req, res) => { 
+    const id = Number(req.params.userId);
+    const user = usersModel.getUser(id);
+    logger.setLogData(user);
+    logger.info('You passed a user ID');
+    res.status(200).json({
+       message: 'You passed a user ID',
+       user
+    });
+};
+
+const createUser = (req, res) => { 
+    let error = {};
+    const user = {
+        userName: req.body.userName,
+        password: req.body.password,
+        firstName: req.body.firstName,
+        lastName: req.body.lastName,
+        address: req.body.address,
+        age: req.body.age,
+        id: idGeneretor.generateId(),
+    };
+    
+    logger.setLogData(user);
+    logger.info("request recieved at /user");
+
+    if (user.userName == null || user.userName == ""){
+        logger.error("username field is empty");
+        error["username"] = "username field is empty";
+    };
+    var userNameValidationRegEx = /^([a-z]|[0-9]|-|_)+$/;
+    var isUserNameValid = userNameValidationRegEx.test(user.userName);
+    if(!isUserNameValid){
+        logger.error("userName field is incorrect");
+		error["userName"] = "userName field is incorrect";
+	} 
+    if (user.password == null || user.password == ""){
+        logger.error("password field is empty");
+        error["password"] = "password field is empty";
+    };
+	if (user.password < 8) {
+		logger.error("password field is incorrect");
+		error["password"] = "password field is incorrect";
+	}
```

Comment:
one check should be enough, you dont want to give hackers any info if they try to still a user's password